cmd/init.go

```diff
@@ -0,0 +1,163 @@
+package cmd
+
+import (
+	"fmt"
+	"path"
+	"text/template"
+
+	"github.com/fatih/color"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"go.k6.io/k6/cmd/state"
+	"go.k6.io/k6/lib/fsext"
+)
+
+const defaultNewScriptName = "script.js"
+
+//nolint:gochecknoglobals
+var defaultNewScriptTemplate = template.Must(template.New("init").Parse(`import http from 'k6/http';
+import { sleep } from 'k6';
+
+export const options = {
+  // A number specifying the number of VUs to run concurrently.
+  vus: 10,
+  // A string specifying the total duration of the test run.
+  duration: '30s',
+
+  // The following section contains configuration options for execution of this
+  // test script in Grafana Cloud.
+  //
+  // See https://grafana.com/docs/grafana-cloud/k6/get-started/run-cloud-tests-from-the-cli/
+  // to learn about authoring and running k6 test scripts in Grafana k6 Cloud.
+  //
+  // cloud: {
```

Comment:
Unfortunately, even if I'd like to have here a `cloud` it's still safer to have this PR with the `ext.loadimpact` :cry:

Reply:
@olegbespalov, could you clarify? My assumption was that this subcommand will land in the same release where we add support for `options.cloud`. I'm fine with changing it to be `options.ext.loadimpact` though and let you change it again in your PRs.

Reply:
Your code will probably be merged first, so producing a working example based on the current state is better. The current state is that there is no such thing `options.cloud`. 

It's not a big deal to rebase the branch that introduces the `options.cloud`  on top of the master after merging this branch (or vice-versa). But the change itself should be atomic.

Reply:
I see the risk that for whatever reason we will end up suggesting the old option. For example, for whatever reason we need to roll back the Cloud option.
What about dividing the current PR in two? One adds the command and one adds the specific Cloud part and it will be merged only when we have the new option.
I'm really against with us having `k6 init` with loadimpact inside.

Reply:
@codebien, not sure what's the risk there. My suggestion was to introduce `k6 init` with `options.ext.loadimpact` and change it to `options.cloud` as a part of @olegbespalov's PR. This way if we decide to roll back the `options.cloud`, it will also revert script sample to use the old version.

Reply:
>it will also revert script sample to use the old version.

That's my point. In that case, we will end by boosting the use of the old option.

Reply:
Right, but if we revert the support for the new one, we should not be suggesting it, right? Anyway, I don't think we will be able to safely revert the config changes anyway, but I see your concern.

Reply:
I meant rollback during the dev cycle.

- We merge here
- We merge the cloud option
- We need to rollback the cloud option
- We release

With the current version, we will end up suggesting the old way. With my suggestion, we will not provide the Cloud block.

Reply:
I see, but even without `options.cloud` we still want to enable users to run their scripts in the cloud.

Reply:
I think promoting the `options.cloud` is a different topic. It's not dependent on the #3348, but rather on the state of our backend, which is the totally opposite thing. So I'd suggest keeping it simple and just accepting in the scope of this PR that only `options.ext.loadimact` exists. 

I see Ivan's point that if we introduce the `k6 init`, it could bring "launch" attention which could produce some scripts that later should be adjusted, but honestly still believe that the amount is less than the ones that are produced from the examples (docs, grafana, etc).

Reply:
It seems you have a consensus on it so I'm fine with you :+1: